lib/containers/common.pm

```diff
@@ -25,13 +26,14 @@ sub is_unreleased_sle {
 }
 
 sub activate_containers_module {
-    my ($running_version, $sp, $host_distri) = get_os_release;
-    my $suseconnect = script_output_retry('SUSEConnect --status-text', timeout => 240, retry => 3, delay => 60);
-    if ($suseconnect !~ m/Containers/) {
-        $running_version eq '12' ? add_suseconnect_product("sle-module-containers", 12) : add_suseconnect_product("sle-module-containers");
-        $suseconnect = script_output_retry('SUSEConnect --status-text', timeout => 240, retry => 3, delay => 60);
+    my $running_version = shift;
```

Comment:
This change will break our test runs. e.g. for https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/containers/common.pm#L48 where the argument is not passed.

Reply:
yes, look at https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/host_configuration.pm#L23
where this test is executed on 3rd party hosts as well.

I would keep same arguments to avoid regressions, and just adapt the code of the suseconnect output parsing.

Reply:
Please re-check.